### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by rituparnadobey-dci**
yes

### CODE REVIEW

**Comment by rituparnadobey-dci** [comment]
yes

**Review by rituparnadobey-dci — COMMENTED** [review]
cool

**Review by rituparnadobey-dci — COMMENTED** [review]
ok

**Review by rituparnadobey-dci — COMMENTED** [review]
done

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
+
+# rockstar-recipes
```
Comment (rituparnadobey-dci) [inline README.md:2]: .....
```suggestion
# rockstar-recipes
```

**Review by rituparnadobey-dci — COMMENTED** [review]
no

File: README.md [hunk 2]
```diff
@@ -1 +1,4 @@
...
 Hi there
+
```
Comment (RituparnaDobey) [inline README.md:4]: ok

**Review by RituparnaDobey — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (RituparnaDobey) [inline README.md:2]: already done

**Review by RituparnaDobey — APPROVED** [review]
yyyy

File: README.md (hunk 2 above)
Comment (RituparnaDobey) [inline README.md:4]: oh yes

**Review by RituparnaDobey — APPROVED** [review]

**Review by RituparnaDobey — APPROVED** [review]
i have done

**Review by RituparnaDobey — APPROVED** [review]
hm

**Pull request closed without merging**